go/logictest/resultparser.go

```diff
@@ -78,6 +100,9 @@ func parseLogEntry(scanner *parser.LineScanner) (*ResultLogEntry, error) {
 
 		// Sample line:
 		// 2019-10-16T12:20:29.0594292-07:00 index/random/10/slt_good_0.test:535: SELECT * FROM tab0 AS cor0 WHERE NULL <> 29 + col0 not ok: Schemas differ. Expected IIIIIII, got IIRTIRT
+
+		// with durations:
+		// 2019-10-16T12:20:29.0594292-07:00 index/random/10/slt_good_0.test:535: SELECT * FROM tab0 AS cor0 WHERE NULL <> 29 + col0 :123456: not ok: Schemas differ. Expected IIIIIII, got IIRTIRT
```

Comment:
Why the colons around the time?

Just put the duration immediately after the finished timestamp, no punctuation, and document its format ("test run duration in ms")